packages/flutter/lib/src/cassowary/solver.dart

```diff
@@ -275,6 +291,7 @@ class Solver {
     return _bulkEdit(variables, applier, undoer);
   }
 
+  /// Attempt to remove the specified edit variable from the solver.
```

Comment:
general comment for all the "remove" methods: add a hyperlink to the corresponding "add" method.